### ISSUE TITLE
Async warning in test

### ISSUE THREAD

**Issue by guimard** [Ajnasz/IniReader#13]
Hello,
new test warns this:
```
-------------- TEST ASYNC --------------
(node:25652) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.
(node:25652) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.
(node:25652) [DEP0013] DeprecationWarning: Calling an asynchronous function without callback is deprecated.
```
This warning breaks heavy tests _(I'm going to package this module for Debian)_

**Comment by Ajnasz** [comment]
Fixed it in v1.2.1

Thanks!

**Comment by guimard** [comment]
Thanks !
